notebooks/advanced/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/2_NTM.ipynb

```diff
@@ -0,0 +1,838 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# News Topic Popularity Forecasting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### This is the second notebook in a series that goes through an example for building an end to end machine learning model for forecasting news topics.  The question we are asking is: \n",
+    "\n",
+    "### Given historical trends on the popularity of a particular topic based on News articles and sentiment, how popular will this topic be in the future?\n",
+    "\n",
+    "The example uses a sample dataset which has been obtained from UCI data repository. The raw dataset can be found here: https://archive.ics.uci.edu/ml/machine-learning-databases/00432/Data/News_Final.csv.\n",
+    "Details of the data preparation process can be found here: https://arxiv.org/pdf/1801.07055.pdf <br/>\n",
+    "\n",
+    "Please first run 1_preprocess.ipynb which runs a series of preprocessing steps on the raw data. This notebook will show you how to build a neural topic model to extract topic vectors from the processed dataset.\n",
+    "\n",
+    "In a subsequent notebook (3_Forecast.ipynb) we will use Amazon Forecast's DeepAR+ Algorithm for time series forecasting to predict the topic's future popularity.\n",
+    "\n",
+    "Note: This notebook works in any python 3 kernel."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import the data for preprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import datetime\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import sys\n",
+    "import time\n",
+    "import re\n",
+    "import boto3\n",
+    "import sagemaker\n",
+    "import nltk\n",
+    "# importing forecast notebook utility from notebooks/ directory\n",
+    "sys.path.insert( 0, os.path.abspath(\"./util\") )\n",
+    "import util\n",
+    "\n",
```

Comment:
Delete Delete "amazon-forecast-samples/notebooks/advanced/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/util", and use "amazon-forecast-samples/notebooks/common/util" instead.